Automattic/woocommerce-payments
# Sending payment_method_update_data param to intent API

Fixes #5951

#### Changes proposed in this Pull Request

This PR requires Automattic/woocommerce-payments-server#4771
Before these PRs we would update the billing address of a saved payment method after the payment was done. This way, when a customer uses a different billing address with a saved payment method it would still use the previous address before being updated. With these changes we update the billing address before processing the payment as it should be.

#### Testing instructions





1. Make sure your server is running the branch from the PR above
1. Place an order saving a new card with a billing address
1. Place an order using the saved card from the previous step but using a new billing address
1. Check the intents in Stripe dashboard and ensure it used the new billing address in the second payment

-------------------


**Post merge**
